synapse/cortex.py

```diff
@@ -124,6 +124,11 @@ async def getModelDict(self):
     async def getModelDefs(self):
         return await self.cell.getModelDefs()
 
+    async def getStormDocs(self):
+        # TODO should this be permission gated in the event that we ever
+        # self-host storm package documentation from inside of the Cortex?
+        return await self.cell.getStormDocs()
```

Comment:
There are other generic "give me information about the cortex" apis which this could fit into